src/app/properties/property-management/availability-pricing/availability-pricing.component.html

```diff
@@ -0,0 +1,41 @@
+<div id="container">
+  <p>AVAILABILITY AND PRICING</p>
+  <div class="grid-container">
+    <mat-card class="demo-inline-calendar-card" color="primary">
+      <mat-calendar [(selected)]="selected"></mat-calendar>
+    </mat-card>
+  </div>
+  <!-- Debug <p>Selected date: {{selected}}</p> -->
+  <br>
+  <div class="grid-item" [formGroup]="infoForm">
+    <mat-form-field class="big-field" appearance="outline" >
+      <mat-label>Price</mat-label>
+      <input matInput type="text" name="price" formControlName="price" />
+    </mat-form-field>
+
+    <div class="price-type-radio">
+      <mat-radio-group aria-label="Select an option">
+        <mat-radio-button value="whole_unit">Price is for whole unit</mat-radio-button>
+        <mat-radio-button value="per_person">Price is per person</mat-radio-button>
+      </mat-radio-group>
+    </div>
+  </div>
+
+  <div class="counter-input">
+    <label for="minimum">Minimum:</label>
+    <div class="counter-logic">
+      <button (click)="decrement('minimum')">
+        <span class="material-icons-outlined">do_not_disturb_on</span>
+      </button>
+      <span>{{ minimumValue }}</span>
+      <button (click)="increment('minimum')">
+        <span class="material-icons-outlined">add_circle_outline</span>
+      </button>
+    </div>
+  </div>
+
+  <section class="example-section">
+    <mat-checkbox class="example-margin">Allow instant booking</mat-checkbox>
+  </section>
```

Comment:
These two elements aren't taken into account when calculating the size of the outermost container.
As a result, this div and section will overlap with whatever is under them.